code/src/java/plugin/experience/gui/ExperienceAdjusterView.java

```diff
@@ -497,12 +497,12 @@ public void valueChanged(ListSelectionEvent e)
 		add(jPanel6);
 	}
 
-	private double calculateRealValue(int i)
+	private static double calculateRealValue(int i)
```

Comment:
Either this method is unused and should be deleted, or you are now violating the access rules for static methods by accessing them from an instance.

Reply:
It was being accessed without an explicit name, so it was allowed. Added `ExperienceAdjusterView.` as a prefix.